drivers/SmartThings/zigbee-fan/config.yml

```
@@ -0,0 +1,6 @@
+name: 'ITM Fan Light 231201'
+packageKey: 'itm-fan-light'
+description: 'EDGE Driver, ITM Fan Light'
+vendorSupportInformation:
+permissions:
+  zigbee: {}
```

since you're creating a new base driver category, I would be as generic in this config as possible (this is basically a copy/paste of z-wave fan's config):
```suggestion
name: 'Zigbee Fan'
packageKey: 'zigbee-fan'
permissions:
  zigbee: {}
description: "SmartThings driver for zigbee fan devices"
vendorSupportInformation: "https://support.smartthings.com"
```

---

It has been modified.